crates/bevy_dev_tools/src/fps_overlay.rs

```diff
@@ -59,21 +85,79 @@ impl Default for FpsOverlayConfig {
                 font_size: 32.0,
                 color: Color::WHITE,
             },
+            frame_time_graph_config: FrameTimeGraphConfig {
```

Comment:
Might be nice to move this into a `Default` implementation on `FrameTimeGraphConfig` itself, since it would let people configuring it to use the `..default()` fill.